Orm/Xtensive.Orm/Core/LockableBase.cs

```diff
@@ -16,35 +16,29 @@ namespace Xtensive.Core
   [Serializable]
   public abstract class LockableBase: ILockable
   {
-    private bool isLocked;
 
     /// <inheritdoc/>
-    public bool IsLocked
+    public bool IsLocked { [DebuggerStepThrough] get; private set; }
+
+    /// <summary>
+    /// Ensures the object is not locked (see <see cref="ILockable.Lock()"/>) yet.
+    /// </summary>
+    /// <exception cref="InstanceIsLockedException">Specified instance is locked.</exception>
+    public void EnsureNotLocked()
     {
-      [DebuggerStepThrough]
-      get { return isLocked; }
+      if (IsLocked) {
+        throw new InstanceIsLockedException(Strings.ExInstanceIsLocked);
+      }
     }
 
+
```

Comment:
What's this extra line for?

Reply:
Just an unnoticed insertion. removed it.